src/js/paste.js

```diff
@@ -129,15 +145,15 @@ var PasteHandler;
                 } else {
                     html = Util.htmlEntities(pastedPlain);
                 }
-                Util.insertHTMLCommand(this.options.ownerDocument, html);
+                Util.insertHTMLCommand(this.ownerDocument, html);
             }
         },
 
         cleanPaste: function (text) {
             var i, elList, workEl,
-                el = Selection.getSelectionElement(this.options.contentWindow),
+                el = Selection.getSelectionElement(this.contentWindow),
```

Comment:
The `initPasteHandler()` function in `core.js` is passing these values into the constructor, so I believe this should be working fine.

However, I do like the chance to rename these options (`this.document` instead of `this.ownerDocument`).  I will do that